_delphi_utils_python/delphi_utils/validator/static.py

```diff
@@ -178,17 +178,17 @@ def check_bad_geo_id_value(self, df_to_test, filename, geo_type, report):
             - report: ValidationReport; report where results are added
         """
         valid_geos = self._get_valid_geo_values(geo_type)
-        unexpected_geos = [geo for geo in df_to_test['geo_id']
-                           if geo.lower() not in valid_geos]
+        unexpected_geos = list({geo for geo in df_to_test['geo_id']
```

Comment:
Made lists like this unique here and in a couple other places. lmk if there's a cleaner way to deduplicate these

Reply:
the set generators in there are great!  you should be able to remove the `list()` wrapped around each of these sets as well; the only other places theyre used is in `len()` calls and f-string replacements, and those will work just as well with as set as they do with a list.